bin/package/android/pom.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>network.mysterium.Node</groupId>
+  <artifactId>Mysterium</artifactId>
+  <version>0.0.0</version>
```

Comment:
groupId: should reflect grouping. And it must be lowercase. Essentially it's like java package - a namespace.
ArtifactId: I would vote for MobileNode.aar - because that's its purpose, and for group id: simply misterium.network, maybe we will produce more artifacts not related to node itself

Reply:
Changed `groupId` and `artifactId`.

Reply:
And sorry for my mistake - it should be mobile-node. No upper cases - words separated by hyphen.

Reply:
fixed